frontend/src/components/common/header/index.css

```diff
@@ -5,6 +5,7 @@
   font-family: "GothamMedium";
   height: auto;
   width: 100%;
+  padding-top: 50px;
```

Comment:
i think a better fix would be to just give this a height of `100px` and then remove [this](https://github.com/the-difference-engine/ymim/blob/staging/frontend/src/components/common/header/index.css#L114-L117)

that decreases the height and gives us a bit more space for the content as well under the navbar

Reply:
thanks for the suggestion, I removed the padding and replace it with height: 100px